scripts/server

```diff
@@ -21,6 +21,10 @@ then
     then
         docker-compose -f "${DIR}/../docker-compose.yml" \
                        -f "${DIR}/../docker-compose.airflow.yml" up
+    elif [ "${1:-}" = "--with-swagger" ]
```

Comment:
Good catch. I took the easy way out by ignoring this case thinking it would be uncommon.